sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -591,3 +591,10 @@
   font-family: "BCSans-bold", "BCSans", "Noto Sans", Verdana, Arial, sans-serif;
   color: $brand-blue;
 }
+
+.restrictions-details {
+  font-style: normal;
```

Comment:
actually this is standard font size and line height that we use in our application. and there are custom classes already built for that. can you double check, 
![image](https://user-images.githubusercontent.com/77353155/196773153-651b9d7a-1d6f-4db5-8cf2-8640b05fb0df.png). and looks like `line-height: 19px;` is not our variable, can you double check with @hellolynn-tbtb about this line height

Reply:
ok. I used the css variables and checked with @hellolynn-tbtb . She said $line-height-header-xsm is ok. Thanks!